src/main/java/ru/testtask/service/ProjectService.java

```diff
@@ -54,5 +59,18 @@ public void updateProject(Project project) {
         return projectRepo.findAll();
     }
 
+    public boolean isCurrentUserOwnerOf(String id){
+        Project project;
+
+            Optional<Project> optionalProject = findProjectById(id);
+
+            if (optionalProject.isPresent()){
+                project = optionalProject.get();
+            }
+            else return false;
+
+           return userService.getCurrentUserId().equals(project.getOwnerId());
```

Comment:
`Objects.equals()` is better because it more safe in case if `userService.getCurrentUserId()` will return null